### ISSUE TITLE
[Bug]: Nord theme lacks visual cues for input fields

### ISSUE THREAD

**Issue by StyraxBenzoin** [0xzerolight/anki_miner#85]
### Description

Nord theme lack visual cues for user input fields e.g. colour contrast, box border

<img width="500" alt="Image" src="https://github.com/user-attachments/assets/7db27ffe-1066-429f-abc2-5f40af0049e4" />

### Steps to reproduce

1. Apply Nord theme

### Anki Miner version

2.6.5

### Operating system

Win 11

### Install method

Windows installer (.exe)

### Logs / error output

```shell
n/a
```

### Additional context

Example implementation in JetBrains IDE

<img height="600" alt="Image" src="https://github.com/user-attachments/assets/fb3cbe38-bc62-4326-a4a7-f76b14923317" />

**Comment by 0xzerolight** [comment]
All done. Nord theme is in my rotation too btw :).
